### PR TITLE
make node integration tests faster

### PR DESCRIPTION
**Pull request by arroyc**

### CODE REVIEW

File: tests/Oryx.Integration.Tests/NodeEndToEndTests.cs [hunk 1]
```diff
@@ -13,137 +13,28 @@
...
+        public readonly string _tempRootDir;
+
+        public NodeEndToEndTestsBase(ITestOutputHelper output, TestTempDirTestFixture testTempDirTestFixture)
         {
             _output = output;
             _hostSamplesDir = Path.Combine(Directory.GetCurrentDirectory(), "SampleApps");
             _tempRootDir = testTempDirTestFixture.RootDirPath;
         }
+    }
+    public class NodeOtherEndtoEndTests : NodeEndToEndTestsBase
```
Comment (kichalla) [inline tests/Oryx.Integration.Tests/NodeEndToEndTests.cs]: nit: blank space before line

File: tests/Oryx.Integration.Tests/NodeEndToEndTests.cs [hunk 2]
```diff
@@ -856,7 +653,7 @@ public async Task CanBuildAndRun_NodeExpressApp_UsingSingleImage_AndCustomScript
                 });
         }
 
-        [Fact(Skip = "#824174: Sync the Node Go startup code with the C# 'run-script' code")]
```
Comment (kichalla) [inline tests/Oryx.Integration.Tests/NodeEndToEndTests.cs]: Do these tests work now?

File: tests/Oryx.Integration.Tests/NodeEndToEndTests.cs [hunk 3]
```diff
@@ -13,137 +13,28 @@
 
 namespace Microsoft.Oryx.Integration.Tests
 {
-    [Trait("category", "node")]
-    public class NodeEndToEndTests : PlatformEndToEndTestsBase
+    public class NodeEndToEndTestsBase : PlatformEndToEndTestsBase
```
Comment (kichalla) [inline tests/Oryx.Integration.Tests/NodeEndToEndTests.cs]: BTW do you plan to even do this similar thing for Node tests present in `Oryx.RuntimeImage.Tests`?

Comment (arroyc) [reply tests/Oryx.Integration.Tests/NodeEndToEndTests.cs]: yes

**Pull request merged**